pkg/volume/secret/secret.go

```diff
@@ -45,8 +45,10 @@ type secretPlugin struct {
 
 var _ volume.VolumePlugin = &secretPlugin{}
 
-var wrappedVolumeSpec = volume.Spec{
-	Volume: &api.Volume{VolumeSource: api.VolumeSource{EmptyDir: &api.EmptyDirVolumeSource{Medium: api.StorageMediumMemory}}},
+func wrappedVolumeSpec() volume.Spec {
```

Comment:
Is it possible to create a common wrappedVolumeSpec method (say in pkg/volume/util.go) and use that in all the 4 locations? (Looks like you may need to pass in a parameter)?

Thanks,
Dims